src/tools/interop/idt/README.md

```diff
@@ -29,7 +29,24 @@ issue uncovered via the manual test. Each ecosystem may implement an analysis
 that analyzes capture data, displays info to the user, probes the local
 environment and generates additional artifacts.
 
-## Getting started
+## Single host installation (no Raspberry Pi)
+
+All features of `idt` are available on macOS and Linux (tested with Debian based systems).  
+If you would prefer to execute capture and discovery from a Raspberry Pi, read the next section instead.  
+
+The machine running `idt` should be connected to the same Wi-Fi network used for testing.  
+Follow the steps below to execute capture and discovery without a Raspberry Pi:  
+
+-   From the parent directory of `idt`, run `source idt/scripts/alias.sh`.
+-   Optionally, run `source idt/scripts/setup_shell.sh` to install aliases
+    permanently.
+-   After `idt` aliases are available in your environment, calling any `idt`
+    command will automatically create a new virtual environment and install
+    python dependencies. 
+    - If you're missing non Python dependencies, you'll be prompted to install them until they're available. 
```

Comment:
```suggestion
    - If you're missing non-Python dependencies, you'll be prompted to install them until they're available. 
```

Reply:
Apologies for not including this earlier; done! I hope I've highlighted all of the substantive changes. Apologies the logging makes things a bit noisy.

Reply:
Will push the suggestion manually in a moment. Have been updating the desc. Resolving this one! Sorry again for not including initially.